aspnetcore/blazor/forms-validation.md

```diff
@@ -390,17 +390,17 @@ private class MyCustomValidator : ValidationAttribute
 
 ### Blazor data annotations validation package
 
-The [Microsoft.AspNetCore.Blazor.DataAnnotations.Validation](https://www.nuget.org/packages/Microsoft.AspNetCore.Blazor.DataAnnotations.Validation) is a package that fills validation experience gaps using the `DataAnnotationsValidator` component. The package is currently *experimental*.
+The [Microsoft.AspNetCore.Components.WebAssembly.DataAnnotations.Validation](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.WebAssembly.DataAnnotations.Validation) is a package that fills validation experience gaps using the `DataAnnotationsValidator` component. The package is currently *experimental*.
```

Comment:
`Microsoft.AspNetCore.Components.WebAssembly.DataAnnotations` should be `Microsoft.AspNetCore.WebAssembly.DataAnnotations`